src/modules/vault/states/useVaultState.ts

```diff
@@ -19,6 +19,8 @@ interface State {
   setBalanceUSD: (balance: string) => void;
   isFirstAssetsLoading: boolean;
   setIsFirstAssetsLoading: (isFirstAssetsLoading: boolean) => void;
+  disableScroll: boolean;
```

Comment:
move this state and toogle control to `src/modules/vault/hooks/create/useCreateVault.ts`